doc/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# build book/apidoc for each tag
+for tag in $(git tag)
+do
+    git checkout $tag
+    mdbook build -d book/$tag
+    cargo doc --target-dir book/api/$tag
```

Comment:
I think it might be better to bundle everything under the version. So we'd have something like:

```sh
    mdbook build -d $tag/book
    cargo doc --target-dir $tag/api
```